sysutils/smart/src/opnsense/service/conf/actions.d/actions_smart.conf

```diff
@@ -1,5 +1,5 @@
 [list]
-command:/bin/ls /dev | grep '^\(ad\|da\|ada\)[0-9]\{1,2\}$'
+command:/usr/local/opnsense/scripts/OPNsense/Smart/list.sh
```

Comment:
wouldn't it make sense to just put it in the same call as the previous by just adding nvme via an or to the list?